### PR TITLE
Readme fixes

### LINKED ISSUE #104: add Travis’ build badge to readme
Add it before Greenkeeper’s badge. You can find the badge at https://travis-ci.org/hoodiehq/first-timers-bot or see other Hoodie repositories for comparison: https://github.com/hoodiehq/hoodie

### PR DESCRIPTION
**Pull request by agonzalez0515**
Closes #104

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +50,9 @@ Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the
 
 
 
-## License
+### License
 
 [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+
+
+Thank you to @michaelmccombie for contributing the pink Hoodie bot. Find him on [Twitter](https://twitter.com/michaelbuilds).
```
Comment (gr2m) [inline README.md]: Maybe add a section `Contributors` or `Thank You` above the License section? There is also a cool project by Kent C. Dodds that you might like for inspiration: https://github.com/kentcdodds/all-contributors

Comment (agonzalez0515) [reply README.md]: Ooh that's really awesome!

Comment (Techforchange) [reply README.md]: @gr2m Omg you read my mind! That's the one I wanted to add :)

File: README.md [hunk 2]
```diff
@@ -48,11 +48,20 @@ First-timers is built with [Probot](https://probot.github.io/).
...
+<!-- Contributors START
+Michael_McCombue michaelmccombie https://twitter.com/michaelbuilds design
+Contributors END -->
+<!-- Contributors table START -->
+| <img src="https://avatars.githubusercontent.com/michaelmccombie?s=100" width="100" alt="Michael McCombue" /><br /><sub>Michael McCombue</sub>](https://twitter.com/michaelbuilds)<br />🎨 |
+| :---: |
+<!-- Contributors table END -->
+This project follows the [all-contributors](https://github.com/kentcdodds/all-contributors) specification.
+
+🎨	Thank you to @michaelmccombie for contributing the pink Hoodie bot. Find him on [Twitter]().
```
Comment (gr2m) [inline README.md]: Add link to twitter account between ()

Comment (agonzalez0515) [reply README.md]: I'm using an Atom package to generate and it has some bugs. So I'm making the fixes manually. Should be fixed now.

File: README.md [hunk 3]
```diff
@@ -46,8 +48,19 @@ Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the
...
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
+### Contributors
+
+Thank you to everyone who has helped with this project.
+<!-- Contributors START
+Michael_McCombie michaelmccombie https://twitter.com/michaelbuilds design
+Contributors END -->
+<!-- Contributors table START -->
+| <img src="https://avatars.githubusercontent.com/michaelmccombie?s=100" width="100" alt="Michael McCombie" /><br />[<sub>Michael McCombie</sub>](https://twitter.com/michaelbuilds)<br />🎨 |
```
Comment (gr2m) [inline README.md]: I think you are supposed to link the 🎨 to the contribution, in that case the logo? In that case I would use this URL: https://raw.githubusercontent.com/hoodiehq/first-timers-bot/51742c62ae3e4e2be7e58d170a9eab73a3871bf4/assets/avatar.png that will work, even if you rename or relocate the image in future, as it points to the specific commit it was added with

File: README.md (hunk 2 above)
Comment (mbad0la) [reply README.md]: Hey @agonzalez0515 !
Just noticed that you are using [all-contributors-atom](https://github.com/o-d-i-n/all-contributors-atom). Thanks for reminding me to release the package (I'm the maintainer 😅 )
I forgot to release after merging the [fix](https://github.com/o-d-i-n/all-contributors-atom/pull/10) for this issue  (calls for automation 🤔 ) ...

PS : It feels weird that something I made could be useful to someone 😛

Comment (agonzalez0515) [reply README.md]: @mbad0la Hi! Awesome! The package is super handy!

**Pull request merged**